lambdas/accesstoken/src/main/java/uk/gov/di/ipv/cri/passport/accesstoken/AccessTokenHandler.java

```diff
@@ -33,24 +34,22 @@
     private final AccessTokenService accessTokenService;
     private final AuthorizationCodeService authorizationCodeService;
 
-    static {
```

Comment:
Do we not need this anymore? Thought we get a conflict with multiple http clients

Reply:
Rather than using this, you can set it like so:

``` ParamManager.getSsmProvider(
                            SsmClient.builder()
                                    .endpointOverride(URI.create(LOCALHOST_URI))
                                    .httpClient(UrlConnectionHttpClient.create())
                                    .region(Region.EU_WEST_2)
                                    .build());```

It's already been set for DynamoDB client, so just needed to set it for the SSMClient.

Reply:
Ah okay nice, that definitely sounds better 👍